README.md

```diff
@@ -15,10 +13,168 @@
+
+**Why is this a new repository?**
+
+We are planning to build more cloud-connected components across multiple frameworks and platforms and it makes more sense to have a unified UI repository with components for all frameworks and platforms.
+
+## Contributing
+
+Right now the best way to contribute is to try out the new Amplify UI and give us feedback. We want to make sure we are building the right tools for you to build amazing applications easily.
+
+## Status
+
+This is a very early preview of the work we are doing. Currently, these packages are not being published to npm yet. We will let you know when they become available. This is currently what we are working on:
+
+| **Component**  | **React** | **Angular** | **Vue** |
+| :------------- | :-------: | :---------: | :-----: |
+| **components** |           |             |
+| Authenticator  |    ✅     |     ✅      |   ✅    |
+| Interactions   |           |             |
+| Storage        |           |             |
+| **primitives** |           |             |
+| Alert          |           |             |
+| Badge          |    ✅     |             |
+| Button         |    ✅     |             |
+| Card           |    ✅     |             |
+| Checkbox       |           |             |
+| Collection     |    ✅     |             |
+| Divider        |    ✅     |             |
+| Dropdown       |           |             |
+| Flex           |    ✅     |             |
+| Grid           |           |             |
+| Icon           |    ✅     |             |
+| Image          |    ✅     |             |
+| Input          |    ✅     |             |
+| Heading        |    ✅     |             |
+| Pagination     |    ✅     |             |
+| Placeholder    |    ✅     |             |
+| RadioButton    |           |             |
+| Slider         |           |             |
+| Spinner        |           |             |
+| Table          |           |             |
+| Tabs           |           |             |
+| Text           |    ✅     |             |
+| TextField      |           |             |
+| Toggle         |           |             |
+| View           |    ✅     |             |
+
+## Looking for other products?
 
-> https://docs.amplify.aws/ui
+- [Amplify Console](https://docs.amplify.aws/console)
+- [Amplify JS](https://docs.amplify.aws/lib/q/platform/js)
+- [Amplify iOS](https://docs.amplify.aws/lib/q/platform/ios)
+- [Amplify Android](https://docs.amplify.aws/lib/q/platform/android)
+- [Amplify Flutter](https://docs.amplify.aws/lib/q/platform/flutter)
+- [Amplify CLI](https://docs.amplify.aws/cli)
+- [Amplify Community](https://amplify.aws/community/)
```

Comment:
cc @mauerbac thoughts on removing the community repo from this list? I'd opt not to include it right now.